slicer-test/opengl/CircleCI_Slicer_Docker.cmake

```diff
@@ -84,7 +84,13 @@ ctest_start( "${SITE_CTEST_MODE}" )
 #ctest_build( BUILD ${CTEST_BINARY_DIR} )
 #    TARGET "${TEST_GUI}" )
 
-ctest_test( BUILD "${CTEST_BINARY_DIRECTORY}" )
+ctest_test( BUILD "${CTEST_BINARY_DIRECTORY}"
+    RETURN_VALUE ctest_return_value )
 #    INCLUDE "${RUN_TEST_GUI}" )
 
 ctest_submit()
+
+if(NOT ctest_return_value EQUAL 0)
+  message( "\n" )
+  message( SEND_ERROR "Some tests have failed. Exit status ${ctest_return_value}" )
```

Comment:
Because I saw that `FATAL_ERROR` interrupt all the process, which means that if we want to do something after testing, it won't be executed. 

It's also because a failing test doesn't matter with what's following so it's not necessary to trigger a `FATAL_ERROR` I guess.